src/ui/svgWidget.js

```diff
@@ -0,0 +1,242 @@
+geo.gui.svgWidget = function (arg) {
```

Comment:
This contains a lot of code from the d3 renderer.  Is this something you are in the process of refactoring?  I would want to think of a way to avoid the code duplication if possible.